src/compton/Compton.hh

```diff
@@ -68,13 +68,22 @@ public:
   explicit Compton(const std::string &filehandle);
 
   //! Constructor to build a multigroup library from an existing pointwise file
+  //! Defaulting the number of angular evals (n_xi) to zero causes the CSK
```

Comment:
Move this text the function documentation block in the `.cc` file.  Try to keep this documentation to a single line.